project-base/app/src/DataFixtures/Demo/PromoCodeDataFixture.php

```diff
@@ -11,6 +11,7 @@
 use Doctrine\Persistence\ObjectManager;
 use Shopsys\FrameworkBundle\Component\DataFixture\AbstractReferenceFixture;
 use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Order\PromoCode\PromoCode as BasePromoCode;
```

Comment:
Base prefix is not necessary. If App PromoCode is already imported, use it, or just remove the alias, please

Reply:
PHPstorm was underlining it as an issue, but im okay with either so i changed it as you recommended.